instat/dlgHistogram.vb

```diff
@@ -112,7 +110,7 @@ Public Class dlgHistogram
         TestOkEnabled()
     End Sub
 
-    Private Sub SetRCodeForControls(bReset As Boolean)
+    Public Sub SetRCodeForControls(bReset As Boolean)

```

Comment:
SetRCodeForControls should be a Public sub since it'll be calling things from it's sub dialogs